widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -826,15 +827,15 @@ private void operatorConnected(String operatorName, String profileImgUrl) {
             items.add(
                     operatorStatusItemIndex,
                     OperatorStatusItem.OperatorFoundStatusItem(
-                            chatState.companyName,
+                            "operatorConnected1",
                             Utils.formatOperatorName(operatorName),
                             profileImgUrl
                     )
             );
         } else {
             items.add(
                     OperatorStatusItem.OperatorFoundStatusItem(
-                            chatState.companyName,
+                            "operatorConnected2",
                             Utils.formatOperatorName(operatorName),
                             profileImgUrl
                     )
```

Comment:
Let me double-check, do you really want to replace `companyName` with constant values `operatorConnected1` and `operatorConnected2`?

Reply:
Left over from debuging. Fixed https://github.com/salemove/android-sdk-widgets/commit/ccca28d0c62f73021106e9e0bd8a900495ccfcb3